docker-local-mojaloop/README.md

```diff
@@ -0,0 +1,389 @@
+# Local Mojaloop + Custom DFSP Test setup
```

Comment:
I don't love the path :`./docker-local-mojaloop`. Mojaloop is implied given the project.

Perhaps this can be `./local-runner`, or `./docker-local`

Reply:
done. went with `./docker-local`